sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -2374,6 +2408,13 @@ private void load(
               parseStatus(jobService.pollJob(jobRef, Bound.LOAD_JOB_POLL_MAX_RETRIES));
           switch (jobStatus) {
             case SUCCEEDED:
+              if (tableDescription != null) {
+                datasetService.patchTableDescription(
```

Comment:
Note to self and reviewers: confirmed that you can't set the table description from the load job itself: https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs#configuration.load